.github/BOTMETA.yml

```diff
@@ -1017,4 +1019,4 @@ macros:
   team_scaleway: QuentinBrosse abarbare jerome-quere kindermoumoute remyleone sieben
   team_solaris: bcoca fishman jasperla jpdasma mator scathatheworm troy2914 xen0l
   team_suse: commel dcermak evrardjp lrupp toabctl AnderEnder alxgu andytom
-  team_virt: joshainglis karmab Aversiste Thulium-Drake
+  team_virt: joshainglis karmab Aversiste Thulium-Drake Ajpantuso
```

Comment:
```suggestion
  team_virt: joshainglis karmab tleguern Thulium-Drake Ajpantuso
```
While we're at it... :) CC @tleguern

Reply:
I can't see who put the thumbs up:) If it was Tristan, feel free to commit and merge:)

Reply:
@Andersson007 it was you ;)

Reply:
hahahah:)

Reply:
I tried to reply by email but I apparently messed it enough to fail. So yes, I renamed my Github account and I have not yet propagated the change where it needs.